test/system/800-scenario.bats

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bats
+#
+# Test for the scenario that
+# "podman import" tarball exported from a container and modified the content.
+#
+
+load helpers
+
+@test "podman import / export" {
+
+  # Create a test file following test
```

Comment:
For consistency with every other test script, would you mind using 4-space indentation?